src/main/scala/ai/privado/tagger/config/PythonDBConfigTagger.scala

```diff
@@ -0,0 +1,90 @@
+package ai.privado.tagger.config
+
+import ai.privado.cache.DatabaseDetailsCache
+import ai.privado.languageEngine.java.language.NodeStarters
+import ai.privado.model.DatabaseDetails
+import io.shiftleft.codepropertygraph.generated.Cpg
+import io.shiftleft.codepropertygraph.generated.nodes.JavaProperty
+import io.shiftleft.passes.ForkJoinParallelCpgPass
+import org.slf4j.LoggerFactory
+import io.shiftleft.semanticcpg.language._
+
+class PythonDBConfigTagger(cpg: Cpg) extends ForkJoinParallelCpgPass[JavaProperty](cpg) {
+
+  private val logger = LoggerFactory.getLogger(getClass)
+
+  override def generateParts(): Array[JavaProperty] = {
+    cpg.property.dedup.toArray
+  }
+
+  override def runOnPart(builder: DiffGraphBuilder, dbUrl: JavaProperty): Unit = {
+    try {
+      if (dbUrl.value.contains("dynamodb")) {
+        parsePropForDynamoDB(dbUrl)
+      } else if (dbUrl.value.contains("postgres:")) {
+        parsePropForPostgreSQL(dbUrl)
+      } else if (dbUrl.value.contains("mongo")) {
+        parsePropForMongoDB(dbUrl)
+      }
+    } catch {
+      case e: Exception => logger.debug("Exception while processing db config: " + e)
+    }
+
+  }
+
+  private def parsePropForDynamoDB(dbUrl: JavaProperty): Unit = {
+    // Example dynamodb url :- https://dynamodb.<region>.amazonaws.com/<table-name>
+    val dbVendor   = "dynamodb"
+    val dbLocation = dbUrl.value.split("\\.")(1)
```

Comment:
what is dbUrl.value.split() doensn't return 2 items, this will result in exception, Handle this gracefully, similarly for other parse functions present